src/object.js

```diff
@@ -43,17 +43,15 @@ _.extend(MarionetteObject.prototype, Backbone.Events, {
   //this is a noop method intended to be overridden by classes that extend from this base
   initialize: function() {},
 
-  destroy: function(options) {
+  destroy: function(...args) {
     if (this._isDestroyed) { return this; }
 
-    options = options || {};
-
-    this.triggerMethod('before:destroy', options);
+    this.triggerMethod.apply(this, ['before:destroy', ...args]);
```

Comment:
I don't know why we'd do the `options = options || {}` thing in this case when it is not that way on view destroy..  however.. you could still support this with ['before:destroy', options, ...args]`  _or_ I _think_ we could be doing `destroy: function(options = {}) {` WHAAAAA?